### PR TITLE
Update counter.markdown

### PR DESCRIPTION
**Pull request by skalavala**
Updated document and removed media references.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

### CODE REVIEW

File: source/_components/counter.markdown [hunk 1]
```diff
@@ -36,28 +36,28 @@ Pick an icon that you can find on [materialdesignicons.com](https://materialdesi
...
 ## {% linkable_title Services %}
 
-### {% linkable_title Media control services %}
+### {% linkable_title counter component services %}
 Available services: `increment`, `decrement`, and `reset`.
 
 #### {% linkable_title Service `counter/increment` %}
 
-Increments the counter with 1 or the given value for the steps.
+Increments the counter value by 1.
```
Comment (fabaff) [inline source/_components/counter.markdown]: See https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/counter.py#L205

**Comment by frenck** [comment]
This PR is running stale. @fabaff and @skalavala anything we can do to move this PR forward?

**Comment by fabaff** [comment]
The component doesn't increment by 1 only but with the value defined in `CONF_STEP` which is 1 by default. The current documentation is right as far as I can tell.

**Comment by fabaff** [comment]
Let's close it. @skalavala feel free to re-open it.

**Pull request closed without merging**